requirements.txt

```diff
@@ -8,7 +8,6 @@ static3==0.5.1
 wsgiref==0.1.2
 -e git+https://github.com/funkybob/django-flatblocks.git#egg=django-flatblocks
 sendgrid-django
-psycopg2==2.5.4
```

Comment:
@brittainhard: 
When I was trying to set up everything locally with @PeterDSteinberg, some dependency error message regarding `psycopg2` pops out. Removing this dependency seems to resolve such issue.